src/modules/launcher/PowerLauncher/SettingsWatcher.cs

```diff
@@ -146,5 +155,20 @@ private static string ConvertHotkey(HotkeySettings hotkey)
             HotkeyModel model = new HotkeyModel(hotkey.Alt, hotkey.Shift, hotkey.Win, hotkey.Ctrl, key);
             return model.ToString();
         }
+
+        protected virtual void Dispose(bool disposing)
+        {
+            if (disposing)
+            {
+                _themeManager?.Dispose();
```

Comment:
Theme manager should not be disposed here. The theme manager is already disposed by the class that initialized it. Can you please update it ?